### DISCUSSION TITLE
xmoj bug: 允许用户查看所有比赛排名

### DISCUSSION THREAD

**Discussion by boomzero** [XMOJ-Script-dev/XMOJ-Script#402] [General]
举例: https://www.xmoj.tech/contestrank.xls.php?cid=1000

https://www.xmoj.tech/contestrank.xls.php?cid=5000

~可以爬下来算rating~

**Comment by langningchen** [comment]
好主意

**Comment by PythonSmall-Q** [comment]
同意，好主意
